### ISSUE TITLE
Link to YouTube playlist and videos for Plasma Hack Week 2021

### ISSUE THREAD

**Issue by namurphy** [PlasmaPy/plasma-hack-week#63]
Videos for the hack week should be posted soon, so we'll need to add that info to the website.

- [ ] Add a link to the YouTube playlist for Plasma Hack Week 2021 in one or more prominent locations
- [ ] Add a link to the video for each tutorial

**Comment by TomNicholas** [comment]
Hey guys - I was wondering if you had a link to the talk I gave at the PlasmaPy hack week 2021 (on how lessons from [pangeo](https://pangeo.io/) could translate to plasma physics software)? I had a look on your YouTube channel but couldn't see it, and would like to include it in a conference abstract.

(Tagging @StanczakDominik too just for visibility)

**Comment by StanczakDominik** [comment]
Darn it. Okay, so we've messed this one up. Most of us have been on pretty hectic schedules ever since the hack week and we haven't yet edited most of the recordings... so they're not on youtube yet. Sorry :(

I can probably try doing a bit of a, heh, hack job on the videos this weekend, prioritizing yours. Ping @rocco8773, let's chat about that on Element.

@TomNicholas what solutions here would make the most sense for you?

**Comment by TomNicholas** [comment]
Hi Dominik, sorry to be a pain. I would just like the video so I can use it as an example of public speaking. Doesn't really have to be edited or uploaded to YouTube or anything - even just a Dropbox link to the file would be great for me. My email is Thomas dot Nicholas at Columbia dot edu

**Comment by StanczakDominik** [comment]
Hey @TomNicholas, fresh off the press, yours is the first one that's uploaded! https://www.youtube.com/watch?v=KYaJrtiF8wo&feature=youtu.be It's a little rough and the HD version is still processing, but I hope it's good enough. I left the post talk discussion in and will mark it with a separate video chapter once I figure out how to manually add them :D but if you'd rather the video didn't include it, I can trim it.

**Comment by TomNicholas** [comment]
This all I needed - thank you so much @StanczakDominik ! Happy for the discussion to be included. You might also want to link to the [accompanying blog post](https://hackmd.io/@TomNicholas/rkyERwcoO) in the video description too.

**Comment by StanczakDominik** [comment]
Yup, I was gonna go through the videos and add details this morning - thanks for making that easier! :D
